.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  extends: ['react-app', 'plugin:@typescript-eslint/recommended'],
+  plugins: ['no-relative-import-paths', 'simple-import-sort'],
+  rules: {
+    '@typescript-eslint/no-empty-function': 'off',
+    '@typescript-eslint/interface-name-prefix': 'off',
+    '@typescript-eslint/camelcase': 'off',
+    '@typescript-eslint/no-inferrable-types': 'off',
+    '@typescript-eslint/explicit-function-return-type': 'off',
+    '@typescript-eslint/no-non-null-assertion': 'off',
+    '@typescript-eslint/no-explicit-any': 'off',
+    '@typescript-eslint/explicit-module-boundary-types': 'off',
+    '@typescript-eslint/ban-ts-comment': 'warn',
+    '@typescript-eslint/ban-types': 'off',
+    'no-relative-import-paths/no-relative-import-paths': ['error', { allowSameFolder: true }],
```

Comment:
> **Motivation:** Copied over from their own [documentation](https://www.npmjs.com/package/eslint-plugin-no-relative-import-paths)
>
> > Moving a file to different folder, could result in changing all imports statement in that file. This will not happen is the import paths are absolute.
>
> Also, the current way of importing leads to stuff like
>
> ```tsx
> import { a } from 'src/.../file'
>
> import { b } from '../file'
> ```
>
> Even though `a` and `b` are defined in the same file, which does not really look nice at best and potentially misleading at worst.

Hmm, given the motivation for this change, is there a reason why the `allowSameFolder` option is set to true? More specifically, why are relative import paths for imported files from the same folder allowed? Moving a file containing such relative import paths will result in these import statements changing.

Reply:
No strong preferences! Just wanted to know if there was a rationale (which you do have) because it would be good to state it somewhere for future reference.